.kitchen.yml

```diff
@@ -1,59 +1,29 @@
 ---
 driver:
-  name: dokken
-  privileged: true  # because Docker and SystemD/Upstart
-  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
-
-transport:
-  name: dokken
-  require_chef_omnibus: <%= ENV['CHEF_VERSION'] || 'latest' %>
+  name: vagrant
```

Comment:
So… I’m confused.

As of d1e970d0b877a23343249d15026393e1ec2651bd (the commit just before #143 landed) Kitchen was only using dokken, and vagrant had been dropped.

I’m good if we want both, though it seems counterproductive to keep going back & forth on this with every pull request.

My preference is to only use dokken, however, as it’s simpler and means that local testing mirrors what the automated testing does.

Reply:
We're using dokken everywhere else, and I'd like to stick to this pattern. Giving people the choice to run it in a proper vm or docker with slimmed down network options is always useful.

Reply:
👍 Dokken doesn't work on windows ... so I need this in order to test lol